docs/static/breaking-changes.asciidoc

```diff
@@ -34,6 +34,15 @@ Here are the breaking changes for 8.0.
 [[bc-core]]
 ==== Changes in Logstash Core
 
+[discrete]
+[[bc-ecs-compatibility]]
+===== ECS compatibility is now on by default
+Many plugins can now be run in a mode that avoids implicit conflict with the <<ecs-ls,Elastic Common Schema>>.
```

Comment:
@karenzone this breaks the docs build, but I don't expect it to because we define `ecs-ls` here: https://github.com/yaauie/logstash/blame/ecs-on-by-default-again/docs/static/ecs-compatibility.asciidoc#L1 -- Am I missing something?

Reply:
Looks reasonable and build fine locally for me. Re-running docs-ci while I investigate further.

Reply:
@elasticmachine run elasticsearch-ci/docs

Reply:
@elasticmachine run elasticsearch-ci/docs

Reply:
Hmmmm... Docs-ci is wonky and not running on command.  Definitely some weirdness going on. Still investigating.

Reply:
@elasticmachine run elasticsearch-ci/docs

Reply:
We've both re-triggered docs-CI several times, and it continues to fail.  When I build locally it works fine, and therefore I can't recreate the issue to troubleshoot it. I agree with your proposal to remove the x-ref for now. We'll be doing more work in this file before release, and will have ample opportunity to add it back.